server/src/test/resources/csars/yaml/valid/lamp-input/my_app/configure_myphpapp.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-CREDENTIALS="/var/www/html/mysql-credentials.php"
+CREDENTIALS="/var/www/html/my_app/mysql-credentials.php"
```

Comment:
Why are these changes to the CSAR necessary?
Did you consult the rest of the team about them?